bot/cogs/nasa.py

```diff
@@ -0,0 +1,167 @@
+import os
+import random
+
+import aiohttp
+
+from discord import Color, Embed
+from discord.ext.commands import (
+    Bot,
+    BucketType,
+    Cog,
+    Context,
+    command,
+    cooldown
+)
+
+NASA_API = os.getenv("NASA_API")
+
+
+def remove_tags(text: str) -> None:
+    import re
+    clean = re.compile('<.*?>')
+    return re.sub(clean, '', text)
+
+
+class Nasa(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+
+    @command()
+    @cooldown(16, 60, BucketType.guild)
+    async def apod(self, ctx: Context) -> None:
+        """Gives you the astronomy picture of the day."""
+        async with self.session.get(f"https://api.nasa.gov/planetary/apod?api_key={NASA_API}") as resp:
+            data = await resp.json()
+
+        if len(data["explanation"]) > 2048:
+            description = f"{data['explanation'][:2045].strip()}..."
+        else:
+            description = data["explanation"]
+        embed = Embed(
+            title=data["title"],
+            description=description,
+            color=Color.blurple()
+        )
+        embed.set_image(url=data["hdurl"])
+        embed.set_footer(text=f"Taken on {data['date']} by {data['copyright']} | Powered by HotWired")
+
+        await ctx.send(embed=embed)
+
+    @command(aliases=["nsearch"])
+    async def nasa_search(self, ctx: Context, *, query: str) -> None:
+        """Search for a query on Nasa."""
+        async with self.session.get(f"https://images-api.nasa.gov/search?q={query}") as resp:
+            data = await resp.json()
+
+        items = data["collection"]["items"]
+        if len(items) > 0:
+            rand_item = random.randint(0, len(items) - 1)
+            item = items[rand_item]
+
+            embed = Embed(
+                title=item["data"][0]["description"],
+                color=Color.blurple()
+            )
+            embed.set_image(url=item["links"][0]["href"])
+            embed.set_footer(
+                text=f"ID: {item['data'][0]['nasa_id']} | Powered by HotWired"
+            )
+
+            await ctx.send(embed=embed)
+        else:
+            await ctx.send("No results found!")
+
+    @command(aliases=["nid"])
+    async def nasa_id(self, ctx: Context, id: str) -> None:
+        """Search for a picture on Nasa with nasa id."""
+        async with self.session.get(f"https://images-api.nasa.gov/asset/{id}") as resp:
+            data = await resp.json()
+
+        try:
+            items = data["collection"]["items"][0]["href"]
+            embed = Embed(
+                color=Color.blurple()
+            )
+            embed.set_image(url=items)
+            embed.set_footer(
+                text=f"ID: {id} | Powered by HotWired"
+            )
```

Comment:
There is no need for this to be multiline
```suggestion
            embed.set_footer(text=f"ID: {id} | Powered by HotWired")
```